fault-proof/bin/challenger.rs

```diff
@@ -122,6 +125,58 @@ where
             .await
     }
 
+    /// Handles claiming bonds from resolved games.
+    pub async fn handle_bond_claiming(&self) -> Result<Action> {
+        let _span = tracing::info_span!("[[Claiming Bonds]]").entered();
+
+        if let Some(game_address) = self
+            .factory
+            .get_oldest_claimable_bond_game_address(
```

Comment:
Where does this function exist on the `DisputeGameFactory`?

I don't see it here: https://github.com/succinctlabs/optimism/blob/develop/packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol?

Reply:
It's an impl of a FactoryTrait! https://github.com/succinctlabs/op-succinct/blob/7db1acc6c5122dcdaabc8454a464ecea6c52de7d/fault-proof/src/lib.rs#L539-L575